lib/zold/version_file.rb

```diff
@@ -0,0 +1,32 @@
+require 'semantic'
+
+module Zold
+  # Read and write .zoldata/version.
+  class VersionFile
+    def initialize(directory)
+      @directory = directory
+    end
+
+    def path
+      File.join(@directory, 'version')
+    end
+
+    def save(version)
+      File.open(path, 'w') do |file|
+        file.puts(version)
+      end
+    end
+
+    def version
+      return unless File.exist?(path)
```

Comment:
@botanicus returning NULL is a code smell: https://www.yegor256.com/2014/05/13/why-null-is-bad.html Never, under any circumstances, use `nil`.